static/css/bq_modal.scss

```diff
@@ -18,6 +18,14 @@
 
 @import "shared/modal.scss";
 
+// TODO (chejennifer): move this back to shared modal scss file.
```

Comment:
super nit - could you add context for what's required to to this?

fwiw, i think it's ok for the modals to go very wide at the smaller screen sizes, even if the content is then surrounded by lots of padding.

Reply:
this was a piece of code that was originally in the shared modal stylesheet and I think you had added it so I don't actually know the context of what it does. The reason I had to move it out of the shared modal stylesheet was that when I was trying to "@forward" and "@use" the different stylesheets, it was complaining about this piece of code ..